resources/prometheus/prometheus-rules.yaml

```diff
@@ -128,6 +128,22 @@ spec:
             summary: "Fleetshard synchronizer container `{{ $labels.pod }}/{{ $labels.container }}` in namespace `{{ $labels.namespace }}` restarted more than 3 times."
             description: "Fleetshard synchronizer container `{{ $labels.pod }}/{{ $labels.container }}` in namespace `{{ $labels.namespace }}` has restarted more than 3 times during the last 30 minutes."
             sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-005-fleetshard-sync-unavailable.md"
+        - alert: RHACSFleetshardCertificateExpiringCritical
+          expr: |
+            ((acs_fleetshard_certificate_expiration_timestamp-time())/60/60/24)<=1
```

Comment:
It would be nice to have this metric deployed before finishing the review. This way it's easier to check the calcuation.